.changeset/many-pumpkins-cry.md

```diff
@@ -0,0 +1,18 @@
+---
+"@latticexyz/store": minor
+"@latticexyz/world": minor
```

Comment:
```suggestion
"@latticexyz/store": major
"@latticexyz/world": major
```
i'd say everything that requires consumers to update something in their code is a `major` change

Reply:
yeah originally had major but then thought just a rename felt less like a major, but happy to go with that